### PR TITLE
+ Added Report Field "Related People" to show a list of people with s…

### PR DESCRIPTION
**Pull request by MrUpsideDown**
…pecified family or known relationships to the subject person.

### CODE REVIEW

**Comment by azturner** [comment]
@MrUpsideDown , a free tool that we use for adding xml comments to public properties/methods is [GhostDoc](http://submain.com/products/ghostdoc.aspx). It is a VS plugin that allows you to right-click a property or method and select "Document This" and it will generate and add xml comments to the property or method.  If you compile in release mode, you'll get warnings for any properties and methods that are missing these comments.

**Comment by MrUpsideDown** [comment]
Ok, thanks. I know the comments are sparse, but my habit is to not add xml comments for overrides unless the override does something unusual. Otherwise, I find the comment is usually just adding bloat by repeating the comment already supplied for the base method. Is the preference to have absolutely all public methods and properties commented? If so, I will be sure to do that.

**Comment by azturner** [comment]
Yep, that's the preference, especially since the compiler by default will generate warnings for any that don't.

**Comment by MrUpsideDown** [comment]
Ok, noted - I'll mend my ways for future commits!

File: Rock/Reporting/DataSelect/Person/RelatedPeopleSelect.cs [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+                                                                                                                        } ) );
+
+            return relatedPeopleQuery;
+        }
+
+        private const string _CtlFormat = "ddlFormat";
+        private const string _CtlKnownRelationshipType = "cblKnownRelationshipType";
+        private const string _CtlFamilyRelationshipType = "cblFamilyRelationshipType";
+
+        public const string FamilyRelationshipParentGuid = "A498E9DB-4BFF-4C52-A6C0-7510EECDF6E7";
```
Comment (dcs619) [inline Rock/Reporting/DataSelect/Person/RelatedPeopleSelect.cs]: @MrUpsideDown Are these existing GUID's in the Rock database or new GUID's that you're creating for the report?

Comment (MrUpsideDown) [inline Rock/Reporting/DataSelect/Person/RelatedPeopleSelect.cs]: @dcs619 These values are only used to persist the settings for this report.

**Pull request merged**